spec/models/rspec_food_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe RecipesFood, type: :model do
+  subject do
+    user = User.create(name: 'Cedric', email: 'Cedric@example.com', password: 'password', confirmed_at: Time.now)
+    recipe = Recipe.create(name: 'Recipe 1', preparation_time: 1, cooking_time: 1, description: 'Recipe 1',
+                           public: true, user:)
+    food = Food.create(name: 'Food 1', measurement_unit: 'KG', price: 20, user:)
+    RecipesFood.new(recipe:, food:, quantity: 5)
+  end
```

Comment:
- [ ] Kindly fix the error. Again this can happen because of the auto-fix. Instead of user: user you can write user: user1

![image](https://user-images.githubusercontent.com/38635158/181916770-e91a8e4f-29d1-4529-b801-2db4958269f5.png)